.travis.yml

```diff
@@ -19,6 +19,8 @@ services:
 before_install:
   - "echo '--colour' > ~/.rspec"
   - "echo 'gem: --no-document' > ~/.gemrc"
+  # solving rainbow issue: https://github.com/sickill/rainbow/issues/48
```

Comment:
I always use present in commits and comments, because its what is actually does and not what you were doing